examples/config/cosmodc2_config.yml

```diff
@@ -136,7 +136,9 @@ TXTwoPointFourier:
     bandwidth: 200
     apodization_size: 0.0
     cache_dir: ./cache
-
+    
+TXRealGaussianCovariance:
```

Comment:
We probably don't want to set this for the default pipeline config - you could make a second config file if you like?

Reply:
Sure makes sense. Will do now. I will just set it to False and whoever wants to run without shape noise can set it to True.

Reply:
Done.